### PR TITLE
Contour labels precision

### PR DESCRIPTION
**Pull request by sankhesh**
Clamp value to 0.0 if it is too small

### CODE REVIEW

File: Rendering/Core/vtkLabeledContourMapper.cxx [hunk 1]
```diff
@@ -603,6 +603,7 @@ bool vtkLabeledContourMapper::PrepareRender(vtkRenderer *ren, vtkActor *act)
       continue;
       }
     metric.Value = scalars->GetComponent(ids[0], 0);
+    metric.Value = std::fabs(metric.Value) > 1e-6 ? metric.Value : 0.0;
```
Comment (durack1) [inline Rendering/Core/vtkLabeledContourMapper.cxx:606]: @sankhesh should this be `std::fabs(metrics.Value) < 1e-6 ? metrics.Value : 0.0;` instead - should the greater than be a less than?

So if less than 1e-6, reset value to 0.0?

**Comment by sankhesh** [comment]
@doutriaux1  @aashish24 
This change fixes the isoline labels precision issue from UV-CDAT/uvcdat#1487
I've updated the baseline in UV-CDAT/uvcdat-testdata@c12229c

File: Rendering/Core/vtkLabeledContourMapper.cxx (hunk 1 above)
Comment (sankhesh) [inline Rendering/Core/vtkLabeledContourMapper.cxx:606]: @durack1 I don't think so. 

If the value is **greater** than a negligible amount (`1e-6`) then keep the value or reset it to `0.0`

Comment (durack1) [inline Rendering/Core/vtkLabeledContourMapper.cxx:606]: @sankhesh apologies - my syntax and logic got confused..

Comment (sankhesh) [inline Rendering/Core/vtkLabeledContourMapper.cxx:606]: @durack1 :coffee: :wink:

Comment (durack1) [inline Rendering/Core/vtkLabeledContourMapper.cxx:606]: @sankhesh should there also be an added test (and rounding) for negligible negative values? Or convert the test value to abs(value)..

Whoops - have I just tripped over my syntax again, is that what `fabs` is doing?

Comment (sankhesh) [inline Rendering/Core/vtkLabeledContourMapper.cxx:606]: Yep. You got it. `fabs` is the `abs` counterpart for floating point values.
